### PR TITLE
KEYCLOAK-11647 Update to 7.3.5

### PR DESCRIPTION
**Pull request by ASzc**

### CODE REVIEW

File: image.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ modules:
           - name: jboss-eap-7-image
             git:
                 url: https://github.com/jboss-container-images/jboss-eap-7-image.git
-                ref: EAP_724_CR2
+                ref: EAP_724_1
```
Comment (iankko) [inline image.yaml]: This will need to wait for the *EAP_725_\** tag yet.

**Review by iankko — CHANGES_REQUESTED** [review]
The tag needs to be updated to _EAP_725_ one once it's available (isn't as of right now AFAICT).

File: image.yaml (hunk 1 above)
Comment (ASzc) [reply image.yaml]: Ok

**Comment by ASzc** [comment]
@iankko updated

**Review by hmlnarik — APPROVED** [review]

**Comment by hmlnarik** [comment]
Thanks @ASzc

**Pull request merged**